mvrx/src/main/kotlin/com/airbnb/mvrx/MavericksView.kt

```diff
@@ -295,4 +300,31 @@ interface MavericksView : LifecycleOwner {
         onFail: (suspend (Throwable) -> Unit)? = null,
         onSuccess: (suspend (T) -> Unit)? = null
     ) = _internalSF(subscriptionLifecycleOwner, asyncProp, deliveryMode, onFail, onSuccess)
+
+    /**
+     * Subscribes to on the given Flow. This can be utilized to create customized subscriptions that for example drop the first
+     * element in the flow before continuing.
+     *
+     * @param deliveryMode If [UniqueOnly], when this [MavericksView] goes from a stopped to start lifecycle a value
+     * will only be emitted if the value has changed. This is useful for transient views that should only
+     * be shown once (toasts, poptarts), or logging. Most other views should use false, as when a view is destroyed
```

Comment:
false is not a valid option, do you mean this?
```suggestion
     * be shown once (toasts, poptarts), or logging. Most other views should use RedeliverOnStart, as when a view is destroyed
```